core-services/ipi-deprovision/gcp.sh

```diff
@@ -24,7 +25,7 @@ logdir="${ARTIFACTS}/deprovision"
 mkdir -p "${logdir}"
 
 
-gce_cluster_age_cutoff="$(TZ=":America/Los_Angeles" date --date="${CLUSTER_TTL}-8 hours" '+%Y-%m-%dT%H:%M%z')"
+gce_cluster_age_cutoff="$(TZ="UTC" date --date="${CLUSTER_TTL}-8 hours" '+%Y-%m-%dT%H:%M%z')"
```

Comment:
There's a reason this was US/Los Angeles.  https://github.com/openshift/ci-tools/pull/399

But forcing this to UTC it's killing clusters only an hour or two into their life